### DISCUSSION TITLE
why no data ?

### DISCUSSION THREAD

**Discussion by hgaong** [apache/skywalking#12538] [Q&A]
![image](https://github.com/user-attachments/assets/1213a3e6-f9b9-44b5-89bd-987c7b239a27)

oap-server error log:
![image](https://github.com/user-attachments/assets/9e56f7de-6280-4206-93f9-7dc43c1781e3)

![image](https://github.com/user-attachments/assets/1aa5de74-b410-4131-9bdf-cc18ef1fe144)

**Comment by wu-sheng** [comment]
The logs you attached are not error logs.
